docs/application/web/api/6.5/device_api/mobile/tizen/ml_single.html

```diff
@@ -0,0 +1,775 @@
+<p><span class="version">Since: </span>
+ 6.5
+            </p>
+<div class="description">
+            <p>
+The <em>ErrorCallback</em> method is launched with these error types:
+            </p>
+            <ul>
+              <li>
+InvalidValuesError - if any of the input parameters contains an invalid value.              </li>
+              <li>
+NotFoundError - if the file with model is not found.              </li>
+              <li>
+NotSupportedError - if the feature is not supported.              </li>
+              <li>
+AbortError - if any other error occurs.              </li>
+            </ul>
+           </div>
+<div class="parameters">
+<p><span class="param">Parameters:</span></p>
+<ul>
+          <li class="param">
+<span class="name">modelPath</span>:
+ Path to a model. Check <a href="filesystem.html">Filesystem API</a> for more information about valid path in Tizen.
+                </li>
+          <li class="param">
+<span class="name">successCallback</span>:
+ The method to call when the model opens successfully.
+                </li>
+          <li class="param">
+<span class="name">errorCallback</span><span class="optional"> [optional]</span><span class="optional"> [nullable]</span>:
+ The method to call when an error occurs.
+                </li>
+          <li class="param">
+<span class="name">inTensorsInfo</span><span class="optional"> [optional]</span><span class="optional"> [nullable]</span>:
+ Input TensorsInfo object. This parameter can change model's input if the model supports flexible input.
+                </li>
+          <li class="param">
+<span class="name">outTensorsInfo</span><span class="optional"> [optional]</span><span class="optional"> [nullable]</span>:
+ Output TensorsInfo object for inference result. This parameter can change model's output if the model supports flexible output.
+                </li>
+          <li class="param">
+<span class="name">fwType</span><span class="optional"> [optional]</span><span class="optional"> [default value = "ANY"]</span>:
+ Type of Neural Network Framework.
+                </li>
+          <li class="param">
+<span class="name">hwType</span><span class="optional"> [optional]</span><span class="optional"> [default value = "ANY"]</span>:
+ Type of hardware resources to be used for NNFWs.
+                </li>
+          <li class="param">
+<span class="name">isDynamicMode</span><span class="optional"> [optional]</span><span class="optional"> [default value = false]</span>:
+ Support Dynamic Mode. Setting this to <var>true</var> enables changing the information of model's input tensors.
+Not all the models or frameworks support changing this information.
+                </li>
```

Comment:
```suggestion
 Support Dynamic Mode: Setting Dynamic Mode to <var>true</var> enables changing the information of model's input tensors. Changing the information of model's input tensors is not supported for all the models or frameworks.
                </li>

Reply:
I make it bold as web api documentation does not support [!NOTE]